src/ensembl/src/content/app/custom-download/state/filters/filtersActions.ts

```diff
@@ -1,29 +1,103 @@
-import { createAction, createStandardAction } from 'typesafe-actions';
+import { ThunkAction } from 'redux-thunk';
+import { ActionCreator, Action } from 'redux';
 
-import customDownloadStorageService from 'src/content/app/custom-download/services/custom-download-storage-service';
 import JSONValue from 'src/shared/types/JSON';
+import { updateActiveConfigurationForGenome } from '../customDownloadActions';
+import {
+  getCustomDownloadActiveGenomeConfiguration,
+  getCustomDownloadActiveGenomeId
+} from '../customDownloadSelectors';
+import set from 'lodash/set';
```

Comment:
to the top of the file, with other imports from node_modules

Reply:
That's done